src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerArgumentBinder.cs

```diff
@@ -85,9 +85,9 @@ public class ControllerArgumentBinder : IControllerArgumentBinder
                 var parameter = parameters[i];
 
                 var result = await BindModelAsync(parameter, controllerContext, valueProvider);
-                if (result != null && result.Value.IsModelSet)
+                if (result != null && result.IsModelSet)
```

Comment:
:+1: - will do a find all references for `.Result` in the product and double check that I didn't miss any.